### PR TITLE
Si/feat/sign up benefit on right hand side

### PR DESCRIPTION
**Pull request by siqbal181**
Very basic for now
Not covering conditional based on login status

### CODE REVIEW

**Review by pablisch — CHANGES_REQUESTED** [review]
Great. Glad you did this! 
Personal opinion, for me there is too much side padding/margin. Too much whitespace in such a thin column. 
Sign up feels like it should be the link to signing up, i.e. an onClick that logs that it is clicked for now, and maybe a bit bigger?

The cursor appears as a text input cursor over the benefits but should turn to pointer over the whole benefit div, and the title if that is to be a link. 

Cursor should change but other comments are just opinions and happy for you to decide on those.

**Comment by siqbal181** [comment]
@pablisch Changed!

**Review by pablisch — CHANGES_REQUESTED** [review]
This looks loads better! I think the benefits content needs to be aligned left personally as depending on width, there can be large misalignments.
Screenshot 2024-04-29 at 11 00 41
Above is how it appeared to me. I think it would be good to pull into main and make decisions on these small things together. See how it looks, maybe even sit with and have ongoing niggles and minor edits lists.

The issue is it doesn't run for me which is confusing as presumably it runs for you. I get all sorts of errors in the feed which stop any of the page from rendering. I notice there is still a Post folder so it must be behind main. Is everything pushed? Do you need to merge main?

**Comment by siqbal181** [comment]
> This looks loads better! I think the benefits content needs to be aligned left personally as depending on width, there can be large misalignments. Screenshot 2024-04-29 at 11 00 41 Above is how it appeared to me. I think it would be good to pull into main and make decisions on these small things together. See how it looks, maybe even sit with and have ongoing niggles and minor edits lists.
> 
> The issue is it doesn't run for me which is confusing as presumably it runs for you. I get all sorts of errors in the feed which stop any of the page from rendering. I notice there is still a Post folder so it must be behind main. Is everything pushed? Do you need to merge main?

Very strange. I've merged main so try now?

> This looks loads better! I think the benefits content needs to be aligned left personally as depending on width, there can be large misalignments. Screenshot 2024-04-29 at 11 00 41 Above is how it appeared to me. I think it would be good to pull into main and make decisions on these small things together. See how it looks, maybe even sit with and have ongoing niggles and minor edits lists.
> 
> The issue is it doesn't run for me which is confusing as presumably it runs for you. I get all sorts of errors in the feed which stop any of the page from rendering. I notice there is still a Post folder so it must be behind main. Is everything pushed? Do you need to merge main?

Very strange. I've merged main and aligned the text to left. Try now?

File: frontend/src/components/SignUpBenefits/SignUpBenefits.tsx [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+
+  const handleSignUpClick = () => {
+    console.log("Right hand - Sign up clicked");
+  };
+
   return (
     <div className="sign-up-benefits-container">
-      Hello. I am the column on the right and nobody cares about me 😞. I'm not
-      even good enough for dummy text.
+      <span className="sign-up-title" onClick={() => handleSignUpClick()}>
```
Comment (pablisch) [inline frontend/src/components/SignUpBenefits/SignUpBenefits.tsx:19]: Small detail. className should probably be sign-up-benefits-title or benefits-title as sign-up-title will almost certainly be needed on the sign-up page. 
I notice on line 31, benefit-text is used but sign-up-benefit-... for most. If brevity is a concern then I think benefits-... would be perfectly reasonable. I don't see any potential clashes with that.

**Review by pablisch — APPROVED** [review]
Great. Mystery over. Looks great.

**Pull request merged**